src/components/Property/TMO.js

```diff
@@ -0,0 +1,18 @@
+import PropTypes from 'prop-types'
+
+const TMO = ({ tmoName }) => {
```

Comment:
TMO is a fairly specific term which I'd like to avoid including in our variable names unless we have to. If we use a more general component name it should be a small readability improvement and we can handle other possible types of details in the future that could be displayed in the same way. E.g. tenure XYZ attribute.

Maybe we could use this:

```
const `TenureDetail = ( { text } ) => {
  // ...
}
```
Then it would be called by parent components something like this:

```
<TenureDetail text=`TMO: ${ tmoName }` />

<TenureDetail text=`XYZ: ${ xyzAttribute }` />
```